app/controllers/groups_controller.rb

```diff
@@ -1,19 +1,20 @@
 class GroupsController < ApplicationController
   def index
-    # currently also using this endpoint to pull down suggested groups
-    groups = if params[:limit].present? # Get a specific amount of trending
-      Group.trending(page: 1, per: params[:limit].to_i)
-    elsif params[:trending].present? # Get all trending groups, paginated
-      Group.trending(page: params[:page].to_i, per: 20)
-    elsif params[:user_id].present? # Get user groups
-      User.find(params[:user_id]).groups.page(params[:page]).per(20)
-    else # Get recent groups
-      Group.order('created_at DESC').take(6)
+    # query values
+    limit = (params[:limit] || 20).to_i
+    page = (params[:page] || 1).to_i
+
+    groups = if params[:trending].present?
+      Group.trending(page: page, per: limit)
+    elsif params[:user_id].present?
+      User.find(params[:user_id]).groups.page(page).per(limit)
```

Comment:
If `User#groups` doesn't have an order applied we should apply one here (`id ASC` would probably suffice) as [kaminari advises](https://github.com/amatsuda/kaminari/wiki/Kaminari-recipes#-dont-forget-to-add-an-order-scope)